staging/src/k8s.io/kubectl/pkg/cmd/exec/exec.go

```diff
@@ -120,10 +120,18 @@ type RemoteExecutor interface {
 type DefaultRemoteExecutor struct{}
 
 func (*DefaultRemoteExecutor) Execute(method string, url *url.URL, config *restclient.Config, stdin io.Reader, stdout, stderr io.Writer, tty bool, terminalSizeQueue remotecommand.TerminalSizeQueue) error {
+	// Legacy SPDY executor is default. If feature gate enabled, fallback
+	// executor attempts websockets first--then SPDY.
 	exec, err := remotecommand.NewSPDYExecutor(config, method, url)
 	if err != nil {
 		return err
 	}
+	if true || cmdutil.RemoteCommandWebsockets.IsEnabled() {
```

Comment:
same question as https://github.com/kubernetes/kubernetes/pull/119186/commits/2f70329b9198b8c60568f4644f6a1b02a649a423#r1264672381

Reply:
Yes--just turning on feature flag to test the `FallbackExecutor`. I will remove the `true` before merge.